.github/workflows/style.yml

```diff
@@ -13,14 +13,11 @@ jobs:
     steps:
       - name: Install Dependencies
         run: |
-          dnf -y install $COMPILER automake libtool pkgconf-pkg-config \
-                autoconf-archive git openssl-devel clang-tools-extra \
+          dnf -y install $COMPILER meson pkgconf-pkg-config \
+                git openssl-devel clang-tools-extra \
```

Comment:
I think that either automake or libtool may have been implicitly be pulling in a c compiler, but meson does not, so you'll have to explicitly now.

Reply:
But here, `$COMPILER` is specified; where does it come from?

Reply:
old remnant, but given we use clang to check style, perhaps we should just install clang instead of gcc as the compiler here.